eng/pipelines/runtime.yml

```diff
@@ -53,6 +53,7 @@ pr:
 
 variables:
   - template: /eng/pipelines/common/variables.yml
+  - template: /eng/common/templates/variables/pool-providers.yml
```

Comment:
you'd need to add this to all pipelines, not just runtime.yml, you can grep for `pipelines/common/variables.yml` to see them.

though now I wonder if we should just add the public pool-providers.yml import into variables.yml so it is imported everywhere and then rely on the templates-official import in runtime-official.yml overriding earlier varlues. Today it doesn't even matter since both .yml have the same value for DncEngInternalBuildPool